docs/concepts/cluster_queue.md

```diff
@@ -35,6 +36,74 @@ This ClusterQueue admits [workloads](workload.md) if and only if:
 
 You can specify the quota as a [quantity](https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/).
 
+## Resources
+
+In a ClusterQueue, you can define quotas for multiple [compute resources](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-types)
+(cpu, memory, GPUs, etc.).
+
+For each resource, you can define quotas for multiple _flavors_. A
+flavor represents different variations of a resource. The variations can be
+defined in a [ResourceFlavor object](#resourceflavor-object).
+
+In a process called [admission](.#admission), Kueue assigns
+[Workload pod sets](workload.md#pod-sets) a flavor for each resource it requests.
```

Comment:
```suggestion
to the [Workload pod sets](workload.md#pod-sets) a flavor for each resource Kueue requests.
```
Not sure if "it" refers to "Kueue" but we need to replace "it" with the object name.